02-express-tutorial/app.js

```diff
@@ -30,10 +30,13 @@ app.get("/", (req, res) => {
 });
 
 
-app.get("/api/v1/products", (req, res) => {
+app.use("/api/v1/products", (req, res) => {
```

Comment:
Hey, this is so close! But we just need to do it for the people route and use the peopleRouter imported about

```
app.use("/api/v1/people", peopleRouter);
```